### PR TITLE
Test: Add molecule scenarios for all examples

### PR DESCRIPTION
**Pull request by ClausHolbechArista**
## Change Summary

Add molecule scenarios for all examples

## Related Issue(s)

Examples did not have the correct output stored as we fixed various issues.

## Component(s) name

Molecule & CI

## Proposed changes

- Add molecule scenarios for each example
- Add new scenarios to refresh-facts
- Add new scenarios to CI
- Remove debug files from l3ls example

## How to test

Tested manually and will also be tested in CI as part of this PR.

### User Checklist

- N/A

### Repository Checklist

- [ ] My code has been rebased from devel before I start
- [ ] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [ ] My change requires a change to the documentation and documentation have been updated accordingly.
- [ ] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by gmuloc — APPROVED** [review]
LGTM

File: ansible_collections/arista/avd/molecule/example-single-dc-l3ls/molecule.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      jinja2_extensions: jinja2.ext.loopcontrols,jinja2.ext.do,jinja2.ext.i18n
+      duplicate_dict_key: error
+  playbooks:
+    converge: '../../examples/single-dc-l3ls/playbook.yml'
+  ansible_args:
+    - --inventory
+    - ${MOLECULE_SCENARIO_DIRECTORY}/../../examples/single-dc-l3ls/inventory.yml
+    # Running with tags build to avoid deploy_eapi to run.
+    - --tags
+    - build
```
Comment (tgodaA) [inline ansible_collections/arista/avd/molecule/example-single-dc-l3ls/molecule.yml:32]: Any reason to add tags only here but not for other examples?

Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/molecule/example-single-dc-l3ls/molecule.yml:32]: This example only has a single playbook, which also includes deployment tasks. The other examples have a separate `build.yml` playbook which only builds the config without deploying it.

**Review by tgodaA — APPROVED** [review]
LGTM

**Pull request merged**